ansible/roles/cli/tasks/download_openwhisk_cli.yml

```diff
@@ -55,13 +55,14 @@
     headers: "{{ headers }}"
     mode: 0755
 
-- name: "unarchive cli ({{ os }}) {{ arc }}"
+- name: "unarchive cli ({{ os }}) {{ arc }} for not linux"
   unarchive:
     src: "{{ cli.nginxdir }}/{{ os }}/{{ arc }}/{{ dest_name }}"
```

Comment:
This PR will not work either. I have forced Linux to use this line to untar, and it did not work either, still not found the path.